src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java

```diff
@@ -1468,17 +1482,18 @@ private void tryPickupEntityItem(@NotNull final EntityItem entityItem)
             final ItemStack itemStack = entityItem.getEntityItem();
 
             final int i = itemStack.stackSize;
-            if (i <= 0 || InventoryUtils.addItemStackToInventory(this.getInventoryCitizen(), itemStack))
+            if (i <= 0 || ItemHandlerHelper.insertItemStacked(this.getInventoryCitizen(), itemStack, true) == null)
             {
+                ItemStack result = ItemHandlerHelper.insertItemStacked(this.getInventoryCitizen(), itemStack, false);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Local variable 'result' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)